scipy/stats/_qmc.py

```diff
@@ -306,6 +311,131 @@ def discrepancy(
                          f" {set(methods)!r}")
 
 
+def geometric_discrepancy(
+        sample: npt.ArrayLike,
+        method: Literal["mindist", "mst"] = "mindist",
+        metric: str = "euclidean") -> float:
+    """Discrepancy of a given sample based on its geometric properties.
+
+    Parameters
+    ----------
+    sample : array_like (n, d)
+        The sample to compute the discrepancy from.
+    method: str, optional
+        The method to use. One of ``mindist`` for minimum distance
+        or ``mst`` for minimum spanning tree.
+    metric: str or callable, optional
+        The distance metric to use. See the documentation
+        for ``scipy.spatial.distance.pdist``.
```

Comment:
Single ticks should resolve the link
```suggestion
        for `scipy.spatial.distance.pdist`.
```